CMakeLists.txt

```diff
@@ -287,6 +287,28 @@ include(CPack)
 include(CPackComponent)
 
 
+set (LIBS
```

Comment:
This is cool and should help us solve all these linking issues we periodically get.

However, I'd suggest renaming it to a longer and less generic name. Maybe something like `AKTUALIZR_EXTERNAL_LIBS`?

Reply:
Do we want all of our executables to link with every library we use? Maybe it's harmless, especially if it is dynamically linked, but that seems like a lot of dependencies for some of the simpler executables.

Reply:
You're right. But I feel that every time we tried to link something with `aktualizr_static_lib` without the full list, it led to some linking errors in the end (sooner or later).

If we stop linking our executables with the full static lib, we'll probably be able to select our external dynamic libraries.